specification/containerservice/resource-manager/readme.md

```diff
@@ -34,15 +34,26 @@ These are the global settings for the ContainerServices API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-06
+tag: package-2019-08
 ```
 
 
+### Tag: package-2019-08
+
+These settings apply only when `--tag=package-2019-08` is specified on the command line.
+
+```yaml $(tag) == 'package-2019-08'
+input-file:
+  - Microsoft.ContainerService/stable/2019-04-30/openShiftManagedClusters.json
+  - Microsoft.ContainerService/stable/2017-07-01/containerService.json
+  - Microsoft.ContainerService/stable/2019-08-01/location.json
+  - Microsoft.ContainerService/stable/2019-08-01/managedClusters.json
+```
 ### Tag: package-2019-06
 
 These settings apply only when `--tag=package-2019-06` is specified on the command line.
 
-```yaml $(tag) == 'package-2019-06'
+``` yaml $(tag) == 'package-2019-06'
```

Comment:
I assume everything still parses ok with this added space?

Reply:
Yes, I can see the parsed file:
https://github.com/Azure/azure-rest-api-specs/blob/2307de018692dde013d3bf3ed0557c6d8f84519d/specification/containerservice/resource-manager/readme.md